### PR TITLE
IO1-1689: fix issue of repeated operators

### PR DESCRIPTION
**Pull request by axelrod-blox**

### CODE REVIEW

File: src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx [hunk 1]
```diff
@@ -72,11 +72,11 @@ const FirstSquare = ({ editPage }: { editPage: boolean }) => {
 
     const response = await Operator.getInstance().getOperators(payload);
     if (response?.pagination?.page > 1) {
-      setOperatorsData([...operatorsData, ...response.operators]);
+      const operators = response.operators.filter((operator: any) => !operatorsData.includes(operator));
+      setOperatorsData([...operatorsData, ...operators]);
```
Comment (meshin-blox) [inline src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx:76]: If each operator here is an object it won't work.
Test this code in a console:

```
const operators = [{id: 1, test: 'foo'}];
operators.includes({id: 1, test: 'foo'})
```

Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx:77]: was added solution with converting to string before include()

File: src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx [hunk 2]
```diff
@@ -72,11 +72,12 @@ const FirstSquare = ({ editPage }: { editPage: boolean }) => {
 
     const response = await Operator.getInstance().getOperators(payload);
     if (response?.pagination?.page > 1) {
-      setOperatorsData([...operatorsData, ...response.operators]);
+      const operatorListInString = operatorsData.map(operator => JSON.stringify(operator));
+      const operators = response.operators.filter((operator: any) => !operatorListInString.includes(JSON.stringify(operator)));
+      setOperatorsData([...operatorsData, ...operators]);
```
[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx:77]: we need all data of operators not only ID

[the remark this answers is not in the document]
Comment (axelrod-blox) [reply src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx:77]: solved

**Review by meshin-blox — APPROVED** [review]

**Pull request merged**